lib/rubocop/cop/rails/pluck_in_where.rb

```diff
@@ -9,27 +9,64 @@ module Rails
       # Since `pluck` is an eager method and hits the database immediately,
       # using `select` helps to avoid additional database queries.
       #
+      # This cop has two different enforcement modes. When the EnforcedStyle
+      # is conservative (the default) then only calls to `pluck` on a constant
+      # (i.e. a model class) in the `where` is used as offenses.
+      #
+      # When the EnforcedStyle is aggressive then all calls to `pluck` in the
+      # `where` is used as offenses. This may lead to false positives
+      # as the cop cannot replace to `select` between calls to `pluck` on an
+      # `ActiveRecord::Relation` instance vs a call to `pluck` on an `Array` instance.
+      #
+      # Autocorrect is disabled by default for this cop since it may generate
+      # false positives.
```

Comment:
Autocorrect is still enabled by default in this PR, did you mean to disable it too? I think changing the default enforced style to `conservative` is probably enough.

Reply:
You're quite right! This sentence is a forgotten copy-paste 💦 I update this PR.
Thank you for your review.